src/network/postgres_protocol_handler.cpp

```diff
@@ -351,6 +360,30 @@ ProcessResult PostgresProtocolHandler::ExecQueryMessage(
   }
 }
 
+ResultType PostgresProtocolHandler::ExecQueryExplain(
+    const std::string &query, parser::ExplainStatement &explain_stmt) {
+  std::unique_ptr<parser::SQLStatementList> unnamed_sql_stmt_list(
+      new parser::SQLStatementList());
+  unnamed_sql_stmt_list->PassInStatement(std::move(explain_stmt.real_sql_stmt));
+  auto stmt = traffic_cop_->PrepareStatement(
+      "explain", query, std::move(unnamed_sql_stmt_list));
+  ResultType status = ResultType::UNKNOWN;
+  if (stmt != nullptr) {
+    traffic_cop_->SetStatement(stmt);
+    std::vector<std::string> plan_info = StringUtil::Split(
+        planner::PlanUtil::GetInfo(stmt->GetPlanTree().get()), '\n');
+    const std::vector<FieldInfo> tuple_descriptor = {
```

Comment:
Curious, why is the brace needed here?

Reply:
I am using brace-initialization here to initialize the first element of the vector to be what's inside the brace. Do you prefer some other way of doing the same?